### PR TITLE
Added Bob Ross Quotes🖌️🎨🖼

### PR DESCRIPTION
**Pull request by UnrealApex**
### Description
Added quotes from arguably the world's greatest painter of all time.

---

### CODE REVIEW

File: static/quotes/english.json [hunk 1]
```diff
@@ -33009,7 +33009,379 @@
...
+      "id": 5610
+    },    
+    {
+      "text": "I got a letter from somebody here a while back, and they said, 'Bob, everything in your world seems to be happy.' That's for sure. That's why I paint. It's because I can create the kind of world that I want, and I can make this world as happy as I want it. Shoot, if you want bad stuff, watch the news.",
+      "source": "Bob Ross",
+      "length": 302,
+      "id": 5687 
+    },
+    {
+      "text": "let's add some happy little trees",
```
Comment (Miodec) [inline static/quotes/english.json:33279]: Capitalisation

File: static/quotes/english.json [hunk 2]
```diff
@@ -33009,7 +33009,379 @@
...
+      "id": 5694
+    },
+    {
+      "text": "All you need to paint is a few tools, a little instruction, and a vision in your mind.",
+      "source": "Bob Ross",
+      "length": 86,
+      "id": 5695
+    },
+    {
+      "text": "wash the brush, just beats the devil out of it",
```
Comment (Miodec) [inline static/quotes/english.json:33327]: Capitalisation

File: static/quotes/english.json [hunk 3]
```diff
@@ -33009,7 +33009,379 @@
...
+    },
+    {
+      "text": "Lets build a happy little cloud.\nLets build some happy little trees.",
+      "source": "Bob Ross",
+      "length": 74,
+      "id": 5701
+    },
+    {
+      "text": "There are no mistakes, just happy accidents.",
+      "source": "",
```
Comment (Miodec) [inline static/quotes/english.json:33364]: No source, duplicate

File: static/quotes/english.json [hunk 4]
```diff
@@ -33009,7 +33009,379 @@
...
+    {
+      "text": "There's nothing in the world that breeds success like success.",
+      "source": "Bob Ross",
+      "length": 62,
+      "id": 5704
+    },
+    {
+      "text": "What can be painted can be punished.",
+      "source": "Bob Ross",
+      "length": 36,
```
Comment (Miodec) [inline static/quotes/english.json:33383]: Too short i would say

File: static/quotes/english.json (hunk 2 above)
Comment (Miodec) [reply static/quotes/english.json:33327]: Also, too short i think

File: static/quotes/english.json [hunk 5]
```diff
@@ -33009,7 +33009,379 @@
...
+    {
+      "text": "Mix up a little more shadow color here, then we can put us a little shadow right in there. See how you can move things around? You have unlimited power on this canvas -- can literally, literally move mountains",
+      "source": "Bob Ross",
+      "length": 209,
+      "id": 5697
+    },
+    {
+      "text": "Believe that you can do it cause you can do it.",
+      "source": "Bob Ross",
+      "length": 47,
```
Comment (Miodec) [inline static/quotes/english.json:33341]: Too short / not quotable enough in my opinion

File: static/quotes/english.json (hunk 1 above)
Comment (Miodec) [reply static/quotes/english.json:33279]: And too short

File: static/quotes/english.json [hunk 6]
```diff
@@ -33009,7 +33009,379 @@
...
+    {
+      "text": "In theory there is no difference between theory and practice. In practice there is.",
+      "source": "Yogi Berra",
+      "length": 83,
+      "id": 5584
+    },
+    {
+      "text": "So many books, so little time.",
+      "source": "Frank Zappa",
+      "length": 30,
```
Comment (Miodec) [inline static/quotes/english.json:33119]: Too short

File: static/quotes/english.json [hunk 7]
```diff
@@ -33009,7 +33009,379 @@
       "text": "Tonight we dance around de flame. Then we get to play de spirit game. Spirit names we shout out loud. Shake de thunder from de spirit cloud. All de songbirds in de tree. Chant a tune to let de spirits free. Den we see dem in de night. Spirits jumping by de firelight",
       "source": "Banana Man, Tally Hall",
       "length": 50,
-      "id": 5567
+      "id":  5567
+    },
+    {
+      "text": "Play is the highest form of research.",
+      "source": "Albert Einstein",
+      "length": 37,
```
Comment (Miodec) [inline static/quotes/english.json:33017]: too short

**Comment by UnrealApex** [comment]
Sure I'll remove the short quotes. For the quotes that aren't Bob Ross, think those might have been from a previous PR. I'll fix that.

**Comment by UnrealApex** [comment]
Being very honest with you, I think quotes under 60 characters are fine as there are more than 150 already there.
Screenshot 2021-08-18 5 21 23 PM

**Comment by UnrealApex** [comment]
Fixed merged conflict, I'll just close this and make the changes in the new one.

**Pull request closed without merging**